apps/frontend/components/FieldTemplate.js

```diff
@@ -0,0 +1,21 @@
+import get from "lodash/get";
```

Comment:
@duranmla I think we can use just one lodash import by using https://github.com/lodash/babel-plugin-lodash and https://github.com/lodash/lodash-webpack-plugin. We can optimize this later.